sdk/identity/azure-identity-extensions/Azure-Database-for-MySQL-README.md

```diff
@@ -7,9 +7,9 @@
       - [Prepare the working environment](#prepare-the-working-environment)
       - [Create an Azure Database for MySQL server](#create-an-azure-database-for-mysql-server)
       - [Configure a firewall rule for your MySQL server](#configure-a-firewall-rule-for-your-mysql-server)
-      - [Enable Azure AD-based authentication](#enable-azure-ad-based-authentication)
+      - [Enable Microsoft Entra-based authentication](#enable-azure-ad-based-authentication)
```

Comment:
Later in the PR you refer to this as `Microsoft Entra ID` - are both acceptable?

Reply:
Great question. See https://learn.microsoft.com/azure/active-directory/fundamentals/new-name#glossary-of-updated-terminology.